ui/pages/swaps/prepare-swap-page/index.scss

```diff
@@ -368,3 +368,9 @@
 @keyframes slide-in {
   100% { transform: translateY(0%); }
 }
+
+.not-enough-balance {
+  &__button {
+    vertical-align: baseline;
+  }
+}
```

Comment:
Shouldn't need these styles now that you are using the actionButton props in BannerAlert